themes/bootstrap3/templates/RecordDriver/DefaultRecord/toolbar.phtml

```diff
@@ -4,38 +4,40 @@
     $this->headScript()->appendFile('https://s7.addthis.com/js/250/addthis_widget.js?pub=' . urlencode($addThis));
   }
 ?>
-<ul class="record-nav nav nav-pills hidden-print">
-  <?php if (count($this->driver->getCitationFormats()) > 0): ?>
-    <li><a class="cite-record" data-lightbox href="<?=$this->recordLink()->getActionUrl($this->driver, 'Cite')?>" rel="nofollow"><i class="fa fa-asterisk" aria-hidden="true"></i> <?=$this->transEsc('Cite this')?></a></li>
-  <?php endif; ?>
-  <?php if ($this->accountCapabilities()->getSmsSetting() !== 'disabled'): ?>
-    <li><a class="sms-record" data-lightbox href="<?=$this->recordLink()->getActionUrl($this->driver, 'SMS')?>" rel="nofollow"><i class="fa fa-mobile" aria-hidden="true"></i> <?=$this->transEsc('Text this')?></a></li>
-  <?php endif; ?>
-  <li><a class="mail-record" data-lightbox href="<?=$this->recordLink()->getActionUrl($this->driver, 'Email')?>" rel="nofollow"><i class="fa fa-envelope" aria-hidden="true"></i> <?=$this->transEsc('Email this')?></a></li>
+<nav class="record-nav">
+  <ul class="action-toolbar">
+    <?php if (count($this->driver->getCitationFormats()) > 0): ?>
+      <li><a class="toolbar-btn" data-lightbox href="<?=$this->recordLink()->getActionUrl($this->driver, 'Cite')?>" rel="nofollow"><i class="fa fa-asterisk" aria-hidden="true"></i> <?=$this->transEsc('Cite this')?></a></li>
```

Comment:
Do we lose anything important by eliminating these more specific class names here? If we definitely do want to eliminate these, is there orphaned LESS somewhere that needs to be cleaned up?

Reply:
No, I looked for JS hooks and LESS styles and came up empty.